cocotb/binary.py

```diff
@@ -281,12 +281,30 @@ def _adjust_twos_comp(self, x):
             rv = x
         return rv
 
-    def get_value(self):
-        """Return the integer representation of the underlying vector."""
+    @property
+    def value(self):
+        "Integer access to the value. **deprecated**"
+        return self.integer
+
+    @value.setter
+    def value(self, val):
+        self.integer = val
+
+    @property
+    def integer(self):
+        "The integer representation of the underlying vector."
         return self._convert_from[self.binaryRepresentation](self._str)
 
-    def get_value_signed(self):
-        """Return the signed integer representation of the underlying vector."""
+    @integer.setter
+    def integer(self, val):
+        self._str = self._convert_to[self.binaryRepresentation](val)
+
+    get_value = integer.fget
+    set_value = integer.fset
+
+    @property
+    def signed_integer(self):
+        "The signed integer representation of the underlying vector."
```

Comment:
```suggestion
        """The signed integer representation of the underlying vector."""
```
While what you have works, it's not conventional to write docstrings with a single set of quotes.